pom.xml

```diff
@@ -1348,6 +1348,7 @@
                  <module>core-java-modules/core-java-networking-3</module>
                  <module>core-java-modules/multimodulemavenproject</module>
                  <module>core-java-modules/core-java-strings</module>
+                 <module>core-java-modules/core-java-httpclient</module>
```

Comment:
FYI @ashleyfrieze I just saw in the docs that there is separate profile for Java 9+ to be run with Jenkins